### PR TITLE
WELD-2658 Fire ProcessBean event for enabled synthetic alternatives.

### PR DESCRIPTION
**Pull request by manovotn**

### CODE REVIEW

File: impl/src/main/java/org/jboss/weld/util/Beans.java [hunk 1]
```diff
@@ -223,16 +224,22 @@ public static boolean containsAllInterceptionBindings(Set<Annotation> expectedBi
...
                     if (enabled.isEnabledAlternativeStereotype(stereotype)) {
-                        return true;
+                        isEnabled = true;
+                        break;
                     }
                 }
-                return false;
             }
+            // For synthetic enabled alternatives, the ModuleEnablement may not yet be aware of them
+            if (!isEnabled && bean instanceof WeldBean && ((WeldBean<?>) bean).getPriority() != null) {
```
Comment (mkouba) [inline impl/src/main/java/org/jboss/weld/util/Beans.java:239]: Hm, how about custom beans implementing `jakarta.enterprise.inject.spi.Prioritized`? I'm just curious ;-)

Comment (manovotn) [reply impl/src/main/java/org/jboss/weld/util/Beans.java:239]: Good point, it is probably affected as well. I'll check it

Comment (manovotn) [reply impl/src/main/java/org/jboss/weld/util/Beans.java]: @mkouba how about now? :)

**Review by mkouba — APPROVED** [review]
Looks good now!

**Pull request merged**